hazelcast-enterprise/Dockerfile

```diff
@@ -30,30 +27,44 @@ ENV HZ_HOME="${HZ_HOME}" \
     CLASSPATH="" \
     JAVA_OPTS=""
 
+LABEL name="Hazelcast IMDG Enterprise" \
+      maintainer="info@hazelcast.com" \
+      vendor="Hazelcast, Inc." \
+      version="${HZ_VERSION}" \
+      release="1" \
+      summary="RHEL UBI based Hazelcast IMDG Enterprise Image" \
```

Comment:
How about changing the summary and description to just: "Hazelcast IMDG Enterprise Image"?